src/com/itacademy/eshop/product/Book.java

```diff
@@ -0,0 +1,17 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class Book extends Product {
```

Comment:
Also not sure if Book abstract class serves any purpose. Besides implemented displayProductInfo method, I can't see any reason why this class should be here, and method implementation doesn't look something specific to Book. I would recommend to drop this class and extend product with Novel directly